### PR TITLE
Add markdownlint step

### PR DESCRIPTION
**Pull request by 0golovatyi**
Add markdownlint step

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,8 +8,10 @@ install:
    - pip install pytest
    - pip install pytest-cov
    - pip install coveralls
+   - npm install -g markdownlint-cli
```
Comment (johng42) [inline .travis.yml]: does this mean we will need to add node.js to install?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -59,14 +59,17 @@ python tests\runtests.py
...
 
-
 ## Documentation Updates
 
 For any process, scripts or API changes documentation needs to be updated accordingly.
 Please use markdown validation tools like web-based[markdownlint](https://dlaa.me/markdownlint/)
 or npm [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli).
 
-TOC for markdown file is built with [markdonw-toc](https://www.npmjs.com/package/markdown-toc).
+TOC for markdown file is built with [markdonw-toc](https://www.npmjs.com/package/markdown-toc):
```
Comment (johng42) [inline CONTRIBUTING.md]: typo
[markdown-toc]

File: README.md [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+
 [![Community Supported](https://img.shields.io/badge/Support%20Level-Community%20Supported-457387.svg)](https://www.tableau.com/support-levels-it-and-developer-tools)
 [![Build Status](https://travis-ci.com/tableau/TabPy.svg?branch=master)](https://travis-ci.com/tableau/TabPy)
 [![Coverage Status](https://coveralls.io/repos/github/tableau/TabPy/badge.svg)](https://coveralls.io/github/tableau/TabPy)
 
 [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/)
 
-TabPy (Tableau Python Server) is external server implementation which allows expanding Tableau with executing Python scripts on table calculation.
+TabPy (Tableau Python Server) is external server implementation which allows
+expanding Tableau with executing Python scripts on table calculation.
```
Comment (johng42) [inline README.md]: typo - the last three words should be
in table calculations.

File: .travis.yml (hunk 1 above)
Comment (0golovatyi) [reply .travis.yml]: yes, node.js is prerequisite for npm

File: CONTRIBUTING.md (hunk 2 above)
Comment (0golovatyi) [reply CONTRIBUTING.md]: fixed

File: README.md (hunk 3 above)
Comment (0golovatyi) [reply README.md]: fixed

**Pull request merged**